.github/workflows/ci.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         # dependencies: ['dev', 'full']
         dependencies: ["full"]
         pydantic-version: ["1.10.9", "2.4.2"]
-        openai-version: ["1.2.4"]
+        openai-version: ["1.30.1"]
```

Comment:
If we only using one version of openai for this it doesn't need to be in the matrix.